config/cable.yml

```diff
@@ -0,0 +1,10 @@
+development:
+  adapter: async
+
+test:
+  adapter: async
+
+production:
+  adapter: redis
+  url: <%= ENV.fetch("REDIS_URL") { "redis://localhost:6379/1" } %>
+  channel_prefix: chord_testbed_production
```

Comment:
Why "chord_testbed_production"? It is not a testbed anymore. I'd suggest a channel name of "earthcube_chords".

Reply:
The cable.yml file was auto-generated by the rails 5.2 upgrate script, and to be honest, I'm not sure why it chose "testbed" as part of the prefix.

 I'm totally fine with using "earthcube_chords_production" for this value.

As a side note, there are a handful of other files that reference "testbed" that we may want to update.

Reply:
Aha, I see the other uses of testbed scattered round CHORDS. Most glaring is application module name in _application.rb_: `module ChordTestbed`. We will leave it as is for this PR. I'll open an issue for this.